tests/acceptance/SupportsApplicationConfig.php

```diff
@@ -2,15 +2,30 @@
 
 namespace tests\acceptance;
 
-use TomPHP\ConfigServiceProvider\ConfigureContainer;
+use TomPHP\ConfigServiceProvider\Configurator;
 
 trait SupportsApplicationConfig
 {
     public function testItAddsConfigToTheContainer()
     {
         $config = ['keyA' => 'valueA'];
 
-        ConfigureContainer::fromArray($this->container, $config);
+        Configurator::apply()->configFromArray($config)->to($this->container);
```

Comment:
@tomphp 

For consistency, should we wrap here already?

```php
Configurator::apply()
    ->configFromArray($config)
    ->to($this->container);
```

An advantage would be that it's easier to see what the _act_ phase looks like, what do you think?